src/main/java/com/hubspot/jinjava/lib/filter/DefaultFilter.java

```diff
@@ -61,7 +64,7 @@ public Object filter(Object object, JinjavaInterpreter interpreter, String... ar
       return object;
     }
 
-    return args[0];
+    return (args[0] instanceof PyWrapper) ? args[0] : Objects.toString(args[0]);
```

Comment:
this is the only real significant change, right?

Reply:
Yes. basically checking if the argument is an instance of PyWrapper class then we do not need to convert it into String.